less/components/amount-buttons.less

```diff
@@ -1,6 +1,17 @@
 .amount-buttons {
   .amount-other-container {
     position: relative;
+    input[name="donation_amount"] + label {
```

Comment:
All changes in non thunderbird less files are me just flattening the styles from newflow to regular.

This can be pattern 3.